docs/reference/integrations.md

```diff
@@ -50,3 +50,18 @@ to the dashboard can be made but will not be persisted upon restart/redeployment
 
 Grafana-Prometheus relate command: `juju relate grafana-k8s:grafana-source prometheus-k8s:grafana-source`  
 Grafana-dashboard relate command: `juju relate wordpress-k8s grafana-dashboard`
+
+### database:
+
+_Interface_: mysql_client
+_Supported charms_: [Charmed MySQL](https://github.com/canonical/mysql-operator)
+
+Database endpoint can be related to mysql based charms, providing long term storage for wordpress.
+Database relation connect wordpress-k8s with charms that support the `my_sql_client` interface on port 3306
+in the database side. See [configuration](https://charmhub.io/wordpress-k8s/configure) for more detail.
```

Comment:
I assume the configurations mentioned here are `mysql-interface-database` and `mysql-interface-user`.
But I can't find any usage related to these two configurations in the code base, which is strange.

Reply:
Yeah, I don't think we should be referencing the configuration here either way. We should only be highlighting the new relation, which doesn't require any configuration.